bids/layout/config/bids.json

```
@@ -69,13 +69,17 @@
         {
             "name": "datatype",
             "pattern": "[/\\\\]+(func|anat|fmap|dwi|meg)[/\\\\]+s"
+        },
+        {
+            "name": "extension",
+            "pattern": "\\.((?:[^_/\\\\]+)+\\.?)$"
```

I think the only "second" extension we permit is `.gz`, and all extensions are alphabetical, at least for now. I'm not sure if things like `surf.gii` will be considered extensions, or just `gii`

```suggestion
            "pattern": "\\.((?:[:Alpha:]+)(?:\\.gz)?)$"
```

Haven't tested this regex thoroughly...